src/Aquarius.ONE.ClientSDK/Common/Configuration/ConfigurationApi.cs

```diff
@@ -314,5 +314,193 @@ public async Task<bool> DeleteConfigurationAsync(string id)
                 throw;
             }
         }
+

```

Comment:
I don't see the ImportNotes endpoint included here is that a separate story?

Reply:
yes, as discussed, it would be another story since it ImportNotes endpoint is not implemented yet

Reply:
I just added the endpoint method for Import Notes